### PR TITLE
[Keyboard] Add keebformom ortho 4x10

### PR DESCRIPTION
**Pull request by nendezkombet**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keebformom/config.h [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0x458F
+#define PRODUCT_ID   0x14E2
+#define DEVICE_VER   0x0001
+#define MANUFACTURER nendezkombet/sandipratama
+#define PRODUCT      Keeb For Mom
```
Comment (drashna) [inline keyboards/keebformom/config.h:26]:
```suggestion
```

File: keyboards/keebformom/config.h [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+ * COLS: AVR pins used for columns, left to right
+ * ROWS: AVR pins used for rows, top to bottom
+ * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
+ *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
+ *
+ */
+/* KFM PCB default pin-out */
+#define MATRIX_ROW_PINS { F5, F6, F7, B1 }
+#define MATRIX_COL_PINS { B3, B2, D1, D0, D4, C6, D7, E6, B4, B5 }
+#define UNUSED_PINS
```
Comment (drashna) [inline keyboards/keebformom/config.h:45]:
```suggestion
```

File: keyboards/keebformom/info.json [hunk 3]
```diff
@@ -0,0 +1,52 @@
+{
+    "keyboard_name": "Keeb For Mom",
+    "url": "",
+    "maintainer": "qmk",
+    "layouts": {
```
Comment (drashna) [inline keyboards/keebformom/info.json:5]:
```suggestion
    "manufacturer": "nendezkombet/sandipratama",
    "usb": {
        "vid": "0x458F",
        "pid": "0x14E2",
        "device_version": "1.0.0"
    },
    "layouts": {
```

File: keyboards/keebformom/readme.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
+## Bootloader
+
+Enter the bootloader in 3 ways:
+
+* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
+* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
+* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```
Comment (drashna) [inline keyboards/keebformom/readme.md:27]:
```suggestion
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `QK_BOOT` if it is available
```

File: keyboards/keebformom/rules.mk [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = no            # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+AUDIO_ENABLE = yes          # Audio output
+LTO_ENABLE = yes
+COMBO_ENABLE = yes
+NO_USB_STARTUP_CHECK = yes  
+ENCODER_ENABLE = no
```
Comment (drashna) [inline keyboards/keebformom/rules.mk:22]:
```suggestion

```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/keebformom/config.h [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+/*
+ * Feature disable options
+ *  These options are also useful to firmware size reduction.
+ */
+
+/* disable action features */
+//#define NO_ACTION_LAYER
+//#define NO_ACTION_TAPPING
+//#define NO_ACTION_ONESHOT
+#define TAPPING_TERM 200
```
Comment (drashna) [inline keyboards/keebformom/config.h:89]:
```suggestion
```

File: keyboards/keebformom/keymaps/default/config.h [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+#ifdef AUDIO_ENABLE
+#define AUDIO_PIN B6
```
Comment (drashna) [inline keyboards/keebformom/keymaps/default/config.h:22]: The audio pin should be defined at the keyboard level.

File: keyboards/keebformom/keymaps/default/config.h [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
```
Comment (drashna) [inline keyboards/keebformom/keymaps/default/config.h:19]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/keebformom/keymaps/default/config.h (hunk 7 above)
Comment (nendezkombet) [reply keyboards/keebformom/keymaps/default/config.h]: fixed, thx  for your correction

**Review by nendezkombet — COMMENTED** [review]
...

**Review by drashna — APPROVED** [review]

File: keyboards/keebformom/rules.mk [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = no            # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no       # Enable keyboard RGB underglow
+AUDIO_ENABLE = no          # Audio output
+LTO_ENABLE = yes
+COMBO_ENABLE = yes
```
Comment (fauxpark) [inline keyboards/keebformom/rules.mk:20]:
```suggestion
```
Combos should only be enabled at keymap level.

File: keyboards/keebformom/config.h [hunk 10]
```diff
@@ -0,0 +1,93 @@
...
+
+/* disable action features */
+//#define NO_ACTION_LAYER
+//#define NO_ACTION_TAPPING
+//#define NO_ACTION_ONESHOT
+
+/* disable these deprecated features by default */
+//#define NO_ACTION_MACRO
+//#define NO_ACTION_FUNCTION
+
```
Comment (fauxpark) [inline keyboards/keebformom/config.h:93]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/keebformom/readme.md [hunk 11]
```diff
@@ -0,0 +1,27 @@
+# Keeb For Mom
+
+![KeebForMom](https://user-images.githubusercontent.com/82454371/181726757-91a95865-3763-4eb9-a22a-bf9bcdbccc05.JPG)
```
Comment (fauxpark) [inline keyboards/keebformom/readme.md:3]: Please rehost this on imgur.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/keebformom/readme.md (hunk 11 above)
Comment (nendezkombet) [reply keyboards/keebformom/readme.md:3]: imgur has been  blocked in my country, can i use other image hosting ?

Comment (nendezkombet) [reply keyboards/keebformom/readme.md]: i used imageshack as image hosting

**Comment by nendezkombet** [comment]
waiting for review

File: keyboards/keebformom/rules.mk (hunk 9 above)
Comment (fauxpark) [reply keyboards/keebformom/rules.mk:20]: Still outstanding.

File: keyboards/keebformom/readme.md [hunk 12]
```diff
@@ -0,0 +1,27 @@
+# Keeb For Mom
+
+![KeebForMom](https://imagizer.imageshack.com/img923/9191/of1evn.jpg)
```
Comment (fauxpark) [inline keyboards/keebformom/readme.md:3]: Please use imgur.

File: keyboards/keebformom/config.h (hunk 10 above)
Comment (fauxpark) [reply keyboards/keebformom/config.h:93]: Still outstanding.

File: keyboards/keebformom/readme.md (hunk 12 above)
Comment (nendezkombet) [reply keyboards/keebformom/readme.md]: image host moved to imgur

File: keyboards/keebformom/config.h [hunk 13]
```diff
@@ -0,0 +1,88 @@
...
+#pragma once
+
+#include "config_common.h"
+
+
+/* key matrix size */
+#define MATRIX_ROWS 4
+#define MATRIX_COLS 10
+
```
Comment (fauxpark) [inline keyboards/keebformom/config.h:25]:
```suggestion
```

File: keyboards/keebformom/config.h [hunk 14]
```diff
@@ -0,0 +1,88 @@
...
+ * fully operational during normal computer usage.
+ *
+ * For a less heavy-handed approach, enable NKRO via magic key (LShift+RShift+N)
+ * or via bootmagic (hold SPACE+N while plugging in the keyboard). Once set by
+ * bootmagic, NKRO mode will always be enabled until it is toggled again during a
+ * power-up.
+ *
+ */
+//#define FORCE_NKRO
+
```
Comment (fauxpark) [inline keyboards/keebformom/config.h:79]:
```suggestion
```

File: keyboards/keebformom/readme.md [hunk 15]
```diff
@@ -0,0 +1,27 @@
+# Keeb For Mom
+
+![KeebForMom](https://i.imgur.com/MvBatHS.jpg)
```
Comment (fauxpark) [inline keyboards/keebformom/readme.md:3]:
```suggestion
![KeebForMom](https://i.imgur.com/MvBatHSh.jpg)
```

File: keyboards/keebformom/info.json [hunk 16]
```diff
@@ -0,0 +1,58 @@
+{
+    "keyboard_name": "Keeb For Mom",
+    "url": "",
+    "maintainer": "qmk",
+    "manufacturer": "nendezkombet/sandipratama",
+    "usb": {
+        "vid": "0x458F",
+        "pid": "0x14E2",
+        "device_version": "1.0.0"
+    },
```
Comment (fauxpark) [inline keyboards/keebformom/info.json:10]:
```suggestion
    },
    "processor": "atmega32u4",
    "bootloader": "caterina",
```

File: keyboards/keebformom/rules.mk [hunk 17]
```diff
@@ -0,0 +1,21 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = caterina
+
```
Comment (fauxpark) [inline keyboards/keebformom/rules.mk:6]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

File: keyboards/keebformom/config.h [hunk 18]
```diff
@@ -0,0 +1,63 @@
...
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
```
Comment (waffle87) [inline keyboards/keebformom/config.h:20]:
```suggestion
```

**Pull request merged**